package.json

```diff
@@ -40,10 +41,12 @@
     "@testing-library/user-event": "7.2.1",
     "babel-eslint": "10.1.0",
     "cross-env": "7.0.2",
+    "cypress": "^5.0.0",
```

Comment:
@rafayet-monon Should we install specific version? 🙌

Reply:
Fixed!